fastlane/Fastfile

```diff
@@ -664,6 +673,41 @@ import "./ScreenshotFastfile"
     )
   end
 
+  #####################################################################################
+  # upload_gutenberg_sourcemaps
+  # -----------------------------------------------------------------------------------
+  # This lane uploads the Gutenberg source maps
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane upload_gutenberg_sourcemaps [internal:<internal>]
+  #
+  # Example:
+  # bundle exec fastlane upload_gutenberg_sourcemaps internal:true
+  #####################################################################################
+  lane :upload_gutenberg_sourcemaps do | options |
+    build_version = ios_get_build_version(internal: options[:internal])
+    
+    gutenberg_bundle = File.join(PROJECT_ROOT_FOLDER, "Pods/Gutenberg/bundle/ios")
+    sourcemaps_folder = File.join(gutenberg_bundle, "sourcemaps")
+    FileUtils.mkdir_p(sourcemaps_folder)
+    # Name of bundle file must be set to main.jsbundle
+    FileUtils.cp(File.join(gutenberg_bundle, 'App.js'), File.join(sourcemaps_folder, 'main.jsbundle'))
+    FileUtils.cp(File.join(gutenberg_bundle, 'App.js.map'), File.join(sourcemaps_folder, 'main.jsbundle.map'))
```

Comment:
It's important that the source map files have specific names, otherwise, Sentry doesn't un-minify the stack traces.